bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/ExpandItem.java

```diff
@@ -539,7 +539,8 @@ private static void handleDPIChange(Widget widget, int newZoom, float scalingFac
 	if (item.height != 0 || item.width != 0) {
 		int newWidth = Math.round(item.width * scalingFactor);
 		int newHeight = Math.round(item.height * scalingFactor);
-		item.setBoundsInPixels(item.x, item.y, newWidth, newHeight, false, true);
+		item.setBoundsInPixels(item.x, item.y, newWidth, newHeight, true, true);
+		item.parent.layoutItems(0, true);
```

Comment:
Why can't we execute this in `ExpandBar#handleDPIChange()`, so that it is only executed once per expand bar instead of updating all updated n times (with n being the number items)?